forms/README.md

```diff
@@ -6,6 +6,7 @@ forms.
 ## Examples
 
 - [pdf_form_add.go](pdf_form_add.go) illustates adding a basic form to a document.
+- [pdf_form_fill_custom_font.go](pdf_form_fill_custom_font.go) replace fonts when generating form.

```

Comment:
maybe reword a bit more like
"illustrates how to specify custom fonts when filling and flattening forms"

Reply:
text updated